src/components/Scenarios/Scenarios.jsx

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import ActionSearch from 'material-ui/svg-icons/action/search';
```

Comment:
I quite like the search icon, we should add it to the buttons in Meta and Explorer too :)